src/main/java/com/ksm/wstbackoffice/entity/Address.java

```diff
@@ -1,17 +1,40 @@
 package com.ksm.wstbackoffice.entity;
 
 import lombok.Data;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "addresses")
```

Comment:
It's a pretty established convention that database table names, in SQL at least, should be singular.

Reply:
done